server/v1/models/accessControl.js

```diff
@@ -4,18 +4,22 @@ const Schema = mongoose.Schema;
 const accessControlSchema = new Schema({
   role: {
     type: String,
+    required: true,
   },
 
   resource: {
     type: String,
+    required: true,
   },
 
   action: {
     type: String,
+    required: true,
   },
 
   attributes: {
     type: String,
+    required: true,
```

Comment:
@dzimmerman902 fyi - this PR sets the properties in the AccessControl collection to required. Otherwise, I was finding there would sometimes be issues in the CLI that would create empty records, and those cause the access control library to break.